features/src/main/feature/camel-features.xml

```diff
@@ -2045,6 +2045,12 @@
         <bundle dependency='true'>wrap:mvn:org.springframework.data/spring-data-redis/${spring-data-redis-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-spring-redis/${project.version}</bundle>
     </feature>
+    <feature name='camel-spring-security' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${camel.osgi.spring.version}'>spring</feature>
+        <bundle dependency='true'>wrap:mvn:org.springframework.security/spring-security-core/${spring-version}</bundle>
```

Comment:
the dependency tree suggests version `6.2.3` and the `spring-version` we use is 6.1.5

Reply:
Yes , you can use ${spring-security-version}